Sample/Sample.properties

```diff
@@ -1,2 +1,5 @@
 url=http://www.flipkart.com
-browser=Chrome
\ No newline at end of file
+browser=Chrome
+changeRequest=testing for first time change
```

Comment:
Mr.Prashant I Think this is an unused code, Please remove it and raise request again.